frontend_tests/zjsunit/index.js

```diff
@@ -82,6 +80,8 @@ test.set_verbose(files.length === 1);
 
 try {
     for (const file of files) {
+        require("../../static/js/templates"); // register Zulip extensions
```

Comment:
This looks good to me.  I assume the speed penalty here is negligible, correct?

Reply:
Yes, I don't see much difference. Using `time`:
Before change:
```
real    0m14.797s
user    0m16.543s
sys     0m0.647s
```
After:
```
real    0m15.721s
user    0m17.538s
sys     0m0.629s
```
(example8.js isn't included in that but that shouldn't affect anything)